tools/ports/boost_headers.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2015 The Emscripten Authors.  All rights reserved.
+# Emscripten is available under two separate licenses, the MIT license and the
+# University of Illinois/NCSA Open Source License.  Both these licenses can be
+# found in the LICENSE file.
+
+import logging
+import os
+import shutil
+
+TAG = 'version_1'
+HASH = '13828c09ebea9dd416a587fbe5e481f626073cb173a137fbcb85044c7bde1249b28af9980da0d2445b8bd1a8387e66a4629103f89031beb90a6c6226c96bc74d'
+
+
+def get(ports, settings, shared):
+  if settings.USE_BOOST_HEADERS != 1:
+    return []
+
+  ports.fetch_project('boost_headers', 'https://github.com/gabrielcuvillier/libboost-includes/archive/' + TAG + '.zip',
```

Comment:
is there not an official boost github repo we could use?

if not we can open one on emscripten-ports

Reply:
AFAIK, there is indeed an official boost github repo (https://github.com/boostorg/boost), but it have to be "build" somehow because the boost headers are not packaged as in a final boost distribution (in the git repo, there is one git sub-repo for each individual boost library). A couple of commands needs to be run first: "boostrap.sh; b2 headers; b2 install".  If it is possible or advisable to run such commands from "python/emscripten ports" infrastructure, then why not. Note that the commands will build intermediate things such as Boost.build and Boost jam (build tools). This is not an instant operation.

Reply:
... or we could simply clone the official boost repo in emscripten-ports, and instead of doing the list of commands to package the headers, we could prepackage them in a release .zip file

Reply:
Yeah, cloning boost might be best. I created a boost repo and team under emscripten-ports now, and added you.